licenses/check_licenses.yml

```diff
@@ -0,0 +1,57 @@
+name: Check and update licenses
```

Comment:
This is still mixing up two PRs, the one for licenses and the one for checking `--module-only`